rst/topics/failover.rst

```diff
@@ -162,6 +162,18 @@ the first appointment). Instead, you should use the promotion API
 the GraphQL ``mutation {cluster{failover_promote()}}``)
 which pushes manual appointments to the state provider.
 
+Stateful failover implies **consistent promotion**: every instance before
+becoming writable performs ``wait_lsn`` operation to sync with the previous one.
```

Comment:
The stateful failover mode implies **consistent promotion**: before becoming writable, each instance  performs the ``wait_lsn`` operation to sync up with the previous one.